usaspending_api/awards/management/commands/loadfinancialassistance.py

```diff
@@ -0,0 +1,189 @@
+import csv
+import logging
+from datetime import datetime
+
+import django
+from django.core.management.base import BaseCommand, CommandError
+
+import usaspending_api.awards.management.commands.helpers as h
+from usaspending_api.awards.models import Award, FinancialAssistanceAward
+from usaspending_api.common.threaded_data_loader import ThreadedDataLoader
+from usaspending_api.references.models import Agency, LegalEntity, Location
+from usaspending_api.submissions.models import SubmissionAttributes
+"""
+
+No PIID in the download from https://www.usaspending.gov/DownloadCenter/Pages/dataarchives.aspx ?
+
```

Comment:
Hmm, the CSVs I've been using always had a FAIN, but I guess I'd better be ready to use either.  Thanks.